configgtm-v1/asmap.go

```diff
@@ -0,0 +1,202 @@
+package configgtm
+
+import (
+	"github.com/akamai/AkamaiOPEN-edgegrid-golang/client-v1"
+)
+
+//
+// Handle Operations on gtm asmaps
+// Based on 1.3 schema
+//
+
+type AsAssignment struct {
+        DatacenterBase                                                    
+        AsNumbers                []int64           `json:"asNumbers"`
+}
+
+// AsMap  represents a GTM AsMap   
+type AsMap struct {
+        DefaultDatacenter        *DatacenterBase   `json:"defaultDatacenter"`
+        Assignments              []*AsAssignment   `json:"assignments,omitempty"`
+        Name                     string            `json:"name,omitempty"`
+        Links                    []*Link           `json:"links,omitempty"`
+}
+
+// NewAsMap creates a new asMap
+func NewAsMap(asmapname string) *AsMap {
+	asmap := &AsMap{Name: asmapname}
+        return asmap
+}
+
+// GetAsMap retrieves a asMap with the given name.                     
+func GetAsMap(asmapname, domainname string) (*AsMap, error) {
```

Comment:
You should use camelCase, so `domainName`

Reply:
corrected